tests/tensorflow/r1.15/common.libsonnet

```diff
@@ -27,8 +27,8 @@ local mixins = import "templates/mixins.libsonnet";
   Convergence:: mixins.Convergence {
     schedule: "0 8 * * 0"
   },
-  # Running convergence tests at 10PM PST daily.
+  # Running functional tests at 10PM PST on Monday, Wednesday..
   Functional:: mixins.Functional {
-    schedule: "0 6 * * *"
```

Comment:
Can we disable these functional tests by default? We're already running the convergence tests, and we don't need the quick feedback while we're not testing RCs.

We can override one or two short tests to run nightly if we want a frequent sanity check. E.g. you can override a schedule like this: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/tensorflow/r1.15/resnet.libsonnet#L56